Chore: (intro to storybook) restructure create addons section

jonniebigodes opened the pull request:
With this pr the intro to storybook create addons section was moved in the toc list.

What was done:
- Applied some minor wording and fixed some typos.
- Fixed the using addons section, by commenting out (for now), the last section as it was introducing issues, due to the fact that #341 was just merged. 


Feel free to provide feedback

domyen commented on content/intro-to-storybook/react/en/creating-addons.md:
```diff
@@ -5,9 +5,9 @@ description: 'Learn how to build your own addons that will super charge your dev
 ---
 
-In the previous chapter we were introduced to one of the key features of Storybook, its robust system of [addons](https://storybook.js.org/addons/introduction/), which can be used to enhance not only yours but also your team's developer experience and workflows.
+Throughout this tutorial we were introduced to one of the key features of Storybook, its robust system of [addons](https://storybook.js.org/addons/introduction/), which can be used to enhance not only yours but also your team's developer experience and workflows.
```
```suggestion
Earlier, we introduced a key Storybook feature: the robust [addons](https://storybook.js.org/addons/introduction/) ecosystem. Addons are used to enhance your developer experience and workflows.
```

jonniebigodes replied:
Changed with the latest commit. Marking it as solved

domyen commented on content/intro-to-storybook/react/en/using-addons.md:
```diff
@@ -158,7 +158,3 @@ If we are using [visual regression testing](/react/en/test/), we will also be in
 ### Merge Changes
 
 Don't forget to merge your changes with git!
-
```
Put all of this content in an `.aside`. Something like "Bonus chapter: Creating your own addon... link to the chapter"

domyen replied:
I don't think it should be removed altogether, it's useful to link to the bonus chapter here

domyen commented:
@jonniebigodes Nice work. Requests for changes above.

Also, can you update the chapter titles so it reads: "Bonus: Create an addon"? That makes it clear what chapters are a part of the narrative and what's a, well, _bonus_.

jonniebigodes commented:
with the latest commit i've changed the titles including the toc ones to adjust to the narrative. Also in the frameworks that have create addons (svelte, react, angular), the using addons chapter last section was reworded and simplified a bit. 

Feel free to provide feedback

domyen commented:
Nice, is the reason we didn't update the react/nl and other languages with the `.aside` because we don't have the correct translations?

![image](https://user-images.githubusercontent.com/263385/86963259-37fbdd00-c132-11ea-9edc-7c457ef19402.png)

jonniebigodes commented:
Yes. I've only covered the toc titles. As the aside, more complex content would be better suited for a native speaker of said language.

domyen commented:
Got it. LGTM 🚀
